From 3a12358c487cfca108b70a69238f9bc620648a5e Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 4 Nov 2020 12:21:41 +0100
Subject: [PATCH v2 1/2] riscv: sort BOARD_SPECIFIC_OPTIONS

Sort implied options in BOARD_SPECIFIC_OPTIONS in the same sequence as in
.config.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
v2:
	new patch
---
 board/emulation/qemu-riscv/Kconfig | 38 +++++++++++++++---------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/board/emulation/qemu-riscv/Kconfig b/board/emulation/qemu-riscv/Kconfig
index 617c4aa33f..36d3b68ab9 100644
--- a/board/emulation/qemu-riscv/Kconfig
+++ b/board/emulation/qemu-riscv/Kconfig
@@ -29,33 +29,33 @@ config BOARD_SPECIFIC_OPTIONS # dummy
 	def_bool y
 	select GENERIC_RISCV
 	select SUPPORT_SPL
-	imply SYS_NS16550
-	imply VIRTIO_MMIO
-	imply VIRTIO_NET
-	imply VIRTIO_BLK
-	imply VIRTIO_PCI
+	imply SMP
+	imply BOARD_LATE_INIT
+	imply SPL_RAM_SUPPORT
+	imply SPL_RAM_DEVICE
+	imply CMD_PCI
+	imply CMD_POWEROFF
 	imply CMD_PING
-	imply CMD_FS_GENERIC
-	imply DOS_PARTITION
-	imply EFI_PARTITION
-	imply ISO_PARTITION
 	imply CMD_EXT2
 	imply CMD_EXT4
 	imply CMD_FAT
-	imply BOARD_LATE_INIT
-	imply SIFIVE_SERIAL
-	imply SMP
+	imply CMD_FS_GENERIC
+	imply DOS_PARTITION
+	imply ISO_PARTITION
+	imply EFI_PARTITION
+	imply E1000
+	imply NVME
 	imply PCI
 	imply DM_PCI
 	imply PCIE_ECAM_GENERIC
-	imply CMD_PCI
-	imply E1000
-	imply NVME
-	imply SPL_RAM_SUPPORT
-	imply SPL_RAM_DEVICE
+	imply SYS_NS16550
+	imply SIFIVE_SERIAL
 	imply SYSRESET
-	imply SYSRESET_SYSCON
-	imply CMD_POWEROFF
 	imply SYSRESET_CMD_POWEROFF
+	imply SYSRESET_SYSCON
+	imply VIRTIO_MMIO
+	imply VIRTIO_PCI
+	imply VIRTIO_NET
+	imply VIRTIO_BLK
 
 endif
-- 
2.28.0

